### ISSUE TITLE
Compatibility with GNOME 44.1

### ISSUE THREAD

**Issue by peter-kehl** [meghprkh/force-quit#20]
Thank you. This has worked well with previous GNOME.

GNOME 44.1 (on current Manjaro Linux) introduces "Disable version validation" (active/applied by default) at https://extensions.gnome.org/local, but this extension doesn't work with it. When I enable Force Quit, and then I refresh https://extensions.gnome.org/extension/770/force-quit, it shows up as disabled. But no obvious error.

Please investigate.

**Comment by peter-kehl** [comment]
Thank you Megh. Somehow it works now (with no apparent change on my side), regardless of "Disable version validation".

**Comment by meghprkh** [comment]
No worries, people have reported a similar issue before but I have been
unable to reproduce it. If possible, could you tell me how you installed
it? And was the issue right after you installed it, in the same login
session?

**Comment by peter-kehl** [comment]
Short story: Unfortunately, I'm not able to reproduce.

Long story:
1. I have had this installed for 1+ year (in older GNOME), it has worked well - thank you.

2. Right after a big update to GNOME 44.1, it (repeatedly) stopped working (after I turned it on, it would turn itself off).

4. Then I've uninstalled a few broken/obsolete GNOME extension, things got better, but I didn't check this yet (I'm sure it stayed disabled). I've installed "krunner" + KDE calendar-like friends, which messed up Super+Space. I've uninstalled "krunner" + KDE calendar-like friends, and suddenly Force Quit works.
